app/controllers/katello/api/rhsm/candlepin_proxies_controller.rb

```diff
@@ -406,15 +395,7 @@ def get_content_view_environments(label = nil, organization = nil)
     end
 
     def system_params
```

Comment:
Should this just be subscription_params or subscription_aspect_params or rhsm_params now? (If we are trying to down play system)